--- a/build.zig
+++ b/build.zig
@@ -481,6 +481,7 @@ fn linkSqlite(b: *Build, mod: *Build.Module, section: bool) void {
     const translate_c = b.addTranslateC(.{
         .root_source_file = lib.getEmittedIncludeTree().path(b, "sqlite3.h"),
         .target = mod.resolved_target.?,
         .optimize = mod.optimize.?,
+        .libc_file = b.libc_file,
     });
     mod.addImport("sqlite3", translate_c.createModule());
 }
@@ -496,6 +497,7 @@ fn linkCurl(b: *Build, mod: *Build.Module, is_tsan: bool, section: bool) void
     const translate_c = b.addTranslateC(.{
         .root_source_file = dep.path("include/curl/curl.h"),
         .target = target,
         .optimize = mod.optimize.?,
+        .libc_file = b.libc_file,
     });
     translate_c.addIncludePath(dep.path("include"));
     mod.addImport("curl", translate_c.createModule());
@@ -989,6 +991,7 @@ fn linkIsocline(b: *Build, mod: *Build.Module) void {
     const translate_c = b.addTranslateC(.{
         .root_source_file = dep.path("include/isocline.h"),
         .target = mod.resolved_target.?,
         .optimize = mod.optimize.?,
+        .libc_file = b.libc_file,
     });
     mod.addImport("isocline", translate_c.createModule());
 }
